src/Core/Config/Util/ConfigFileTransformer.php

```diff
@@ -26,67 +26,191 @@
  */
 class ConfigFileTransformer
 {
+	/**
+	 * The public method to start the encoding
```

Comment:
This doesn't say anything that the prototype of the method already says. What more can you say about this method? When is it supposed to be used? Caveats?